app/api_topology.go

```diff
@@ -30,10 +30,19 @@ type APIEdge struct {
 	Metadata report.AggregateMetadata `json:"metadata"`
 }
 
+func render(rpt report.Report, maps []topologyMapper) report.RenderableNodes {
+	result := report.RenderableNodes{}
+	for _, m := range maps {
+		rns := m.selector(rpt).RenderBy(m.mapper, m.pseudo)
+		result.Merge(rns)
+	}
+	return result
+}
```

Comment:
Just so I've got it straight in my head — this is the meat of the change, right? This is the bit that flattens all of the metadatas into each other?